src/app/layout/qgslayoutmanagerdialog.cpp

```diff
@@ -63,7 +65,9 @@ QgsLayoutManagerDialog::QgsLayoutManagerDialog( QWidget *parent, Qt::WindowFlags
 
```

Comment:
I realize that it's not in this PR but I'd like to see more `QgsSettings::Sections` instead of hardcoded prefixes like `UI`

Reply:
Fixed